frontend/src/components/Buttons/SegmentedButtons/index.tsx

```diff
@@ -17,9 +17,13 @@ interface SegmentedButtonsProps {
 export const SegmentedButtons: React.FC<SegmentedButtonsProps> = ({
     actions,
 }) => {
+    const length = actions.length;
+
     return (
         <div className="inline-block">
-            <div className="grid grid-cols-3">
+            <div
+                className={`grid  grid-cols-[repeat(${length},minmax(0px,1fr))]`}
```

Comment:
Did you make sure this is valid Tailwind?
https://tailwindcss.com/docs/content-configuration#dynamic-class-names

Reply:
for me it looks like this:
![image](https://user-images.githubusercontent.com/116296251/213437852-8a89c104-4546-4f46-83a7-40fa8e08769a.png)
but i imagine it should look like this (grid-cols-2):
![image](https://user-images.githubusercontent.com/116296251/213437946-b7ca25fe-e55b-4217-a40a-df34a15b451c.png)

Reply:
Weird, I did not see that happening when I run the program. I'll just make it then fixed to two rather than try to put calculated value to the tailwind.